lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -656,6 +698,8 @@ data TxSkeleton = TxSkeleton
     , txInputCount :: !Int
     , txOutputs :: ![TxOut]
     , txChange :: ![Set AssetId]
+    , txScripts :: [Script KeyHash]
+    , txMintAssets :: [AssetId]
```

Comment:
I'm guessing the skeleton would also need to include the assets that are burned? (As these take space in the transaction, so they'd need to be considered when calculating a transaction's size in the `estimateTxSize` function.)